API-strategie-modulen/API-strategie-mod-geo/crs.md

```diff
@@ -138,7 +138,7 @@ In an API that supports the creation of items, POST requests with geospatial con
   <p>In a request (i.e. when creating an item on the server):</p>
   <uL>
     <li>Issue an HTTP POST request to the API with spatial data in the request body, including the <code>Content-Crs</code> header with the value of the CRS identifier for the spatial data in the body.</li>
-    <li>Verify that a document was returned with a status code 201.</li>
+    <li>Verify that a document was returned with status code <code>201</code> in case a new item was created, or with status code <code>200</code>.</li>
```

Comment:
Als er bij het creëren van een resource een 201 response wordt gegeven, dan behoort (volgens RFC 7231) de response de locatie te bevatten waar de gecreëerde resources gevonden kan worden. Als er een 200 response wordt gegeven, dan behoort de response de gecreëerde resource zelf te bevatten. Het 'document' zal er dan in beide gevallen anders uitzien. Willen we dat onderscheid hier duidelijk maken?

Reply:
Ik vind dat het hier voldoende is zoals @sweco-nlgemo het nu heeft opgeschreven. Het werken met status codes is niet geo-specifiek en het wordt al beschreven in [API Design Rules extensions](https://docs.geostandaarden.nl/api/API-Strategie-ext/#http-status-codes)

Reply:
Akkoord.